PendingReleaseNotes

```diff
@@ -1,5 +1,11 @@
 15.2.8
 ------
+* mgr-pg_autoscaler: Autoscaler will now start out with scaling each
+  pools to have a full complements of pgs from the start and will only
+  decrease it when pools need more pgs due to increased usage. The solves
```

Comment:
other pools

Reply:
How about reframing the last sentence? Maybe something like "This improves out of the box performance of Ceph by allowing more PGs to be created for a given pool."